cpp/oneapi/dal/algo/basic_statistics/test/batch.cpp

```diff
@@ -63,4 +63,21 @@ TEMPLATE_LIST_TEST_M(basic_statistics_batch_test,
     this->general_checks(data, weights, compute_mode);
 }
 
+TEMPLATE_LIST_TEST_M(basic_statistics_batch_test,
+                     "basic_statistics common CSR flow",
+                     "[basic_statistics][integration][batch]",
+                     basic_statistics_sparse_types) {
+    SKIP_IF(this->not_float64_friendly());
+    const auto data = GENERATE_COPY(te::csr_table_builder(5, 5),
+                                    te::csr_table_builder(7, 10),
+                                    te::csr_table_builder(100, 100));
```

Comment:
Please add the data that have the dimensions bigger than maximal workgroup size on the devices we are using. 100 might be not enough I think.